### PR TITLE
DOCS: Add finetune_depth to tutorial on improving accuracy

### PR DESCRIPTION
**Pull request by marcopeix**
Showcase the `finetune_depth` parameter which controls how many parameters in the model get fine-tuned in the tutorial on improving forecast accuracy with TimeGPT.

### CODE REVIEW

**Review by AzulGarza — APPROVED** [review]
thank you, @marcopeix! lgtm:)

it's interesting that in this case fine-tuning more parameters leads to similar performance than adding exogenous variables.
